app/controllers/posts_controller.rb

```diff
@@ -1,10 +1,10 @@
-class PostsController < ApplicationController
-  def index
-    user = params[:user_id]
-    @post = Post.where(user_id: user)
+class UsersController < ApplicationController
+  def show
+    @user = User.find(params[:id])
+    @posts = @user.recent_post
   end
 
-  def show
-    @post = Post.find_by(user_id: params[:user_id])
+  def index
+    @users = User.all
   end
 end
```

Comment:
- [ ] While you have made a great effort in this milestone, you have put the `PostsController` class in the `users_controller.rb` file which might cause some issues as Rails rely on these naming conventions to associate the controller to the correct view file.

Reply:
@AnselemOdims 
Thank you so much for picking up that error. I have rectified the mistake at this [commit.](https://github.com/Lucash2022/Blog_App/pull/9/commits/d23213d1f2cc919bcdc73fb5aeec3d73fe3d460e)